src/proto/beats.proto

```diff
@@ -31,12 +50,13 @@ message Beat {
   optional double frame_position = 1 [ default = 0 ];
```

Comment:
We should have either Beats or BpmMarkers. Storing both is redundant and a likely source of bugs.

Reply:
We only have the LegacyBeat for old types. We are only using BpmMarkers now.